.github/workflows/publish-dev-docs.yml

```diff
@@ -14,6 +14,10 @@ jobs:
   copy-file:
     runs-on: ubuntu-latest
     steps:
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
```

Comment:
I think this should be specified in root package.json file so workflow can make use of it.